GrandCentralBoard/Widgets/Harvest/HarvestSource.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  HarvestSource.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-11.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import GrandCentralBoardCore
+
+
+class HarvestSource: Asynchronous {
```

Comment:
`final class`

I would suggest `HarvestSource : Asynchronous` to visually separate inheritance from type.